contracts/views/LiquityView.sol

```diff
@@ -222,4 +222,30 @@ contract LiquityView is LiquityHelper, DSMath {
             next = SortedTroves.getPrev(next);
         }
     }
+
+    /// @notice Returns the number of troves based on the targetDebtInFront
+    /// @param _acc Accumulated sum used in subsequent calls, 0 for first call
+    /// @param _iterations Maximum number of troves to traverse
+    /// @param _targetDebtInFront Lusd debt in wei
+    /// @return next Trove owner address to be used in the subsequent call
```

Comment:
There is no input parameter which takes this into account, if trove list is too long to not be passed through in one tx, there would be no way to pass through the whole list